regnety/dataset/imagenet.py

```diff
@@ -37,8 +36,8 @@ class ImageNet:
     def __init__(
         self,
         tfrecs_filepath: List[str]  = None,
-        batch_size: int = 128,
-        image_size: int = 224,
+        batch_size: int = 1024,
+        image_size: int = 512,
         augment_fn: Union[str, Callable]  = "default",
         num_classes: int = 10,
         randaugment: bool = True,
```

Comment:
`num_classes=10`? Also, we are NOT using the original RandAugment now, should this argument be named as it is? Or is it there to allow the user to incorporate it?

Reply:
I'll remove the randaugment argument. No, the user doesn't get the option to implement original RandAugment. 

If augment_fn == 'default' , then we'll apply our augmentations.